src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx

```diff
@@ -4,17 +4,48 @@ import PropTypes from "prop-types";
 import { Button, Tooltip } from "antd";
 import Columns from "./Columns/Columns";
 
+import styled from "styled-components";
+import { grey2, grey5, grey9 } from "../../../../../styles/colors";
+
+const ControlPanel = styled.div`
+  height: 100%;
+  position: relative;
+  overflow: hidden;
+  background-color: ${grey2};
+  border-left: 1px solid ${grey5};
```

Comment:
For border-left should we maybe create a const in a styles/borders.js file?

Reply:
Updated in a5a438902cda0f715ed0fb97d8d416e97eb613ec